draft-ietf-tls-tls13.md

```diff
@@ -1346,39 +1357,37 @@ following four sets of options in its ClientHello:
   (EC)DHE shares for some or all of these groups.
 - A "signature_algorithms" ({{signature-algorithms}}) extension which indicates the signature
   algorithms which the client can accept.
-- A "pre_shared_key" ({{pre-shared-key-extension}}) extension which contains the identities
-  of symmetric keys known to the client and the key exchange
-  modes which each PSK supports.
+- A "pre_shared_key" ({{pre-shared-key-extension}}) extension which contains a symmetric
+  keys known to the client and a "psk_key_exchange_modes" ({{pre-shared-key-exchange-modes}}) extension which indicates
+  the key exchange modes that may be used with PSKs.
 
-If the server does not select a PSK, then the first three of these
+If the server does not select PSK, then the first three of these
 options are entirely orthogonal: the server independently selects a
 cipher suite, an (EC)DHE group and key share for key establishment,
 and a signature algorithm/certificate pair to authenticate itself to
 the client. If there is overlap in the "supported_group" extension
 but the client did not offer a compatible "key_share" extension,
 then the server will respond with a HelloRetryRequest ({{hello-retry-request}}) message.
 
-If the server selects a PSK, then the PSK will indicate which key
-establishment modes it can be used with (PSK alone or with (EC)DHE) and
-which authentication modes it can be used with (PSK alone or PSK with
-signatures). The server can then select those key establishment and
-authentication parameters to be consistent both with the PSK and the
+If the server selects a PSK, then it MUST only be used with
```

Comment:
Should probably be consistent about "PSK" vs "a PSK" (vs. "a PSK mode"?) here and L1364